src/main/java/pl/nogacz/snake/application/LoadGame.java

```diff
@@ -0,0 +1,59 @@
+package pl.nogacz.snake.application;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.ObjectInputStream;
+
+import pl.nogacz.snake.board.Board;
+import pl.nogacz.snake.board.BoardInfo;
+
+public class LoadGame {
+
+    Board board;
+
+    public LoadGame(Board board){
+
+        this.board = board;
+    }
+
+    public boolean startLoad(){
+
+        FileInputStream fi;
+        ObjectInputStream in;
+
+        String[] fileNames;
+        File f = new File("src\\main\\SavedGames");
+
+        fileNames = f.list();
```

Comment:
Her satırdan sonra rastgele çift newline koyma. Bunlara gerek yok. Bir kod bloğu ile başka bir iş yapan kod bloğunu okunaklılığı iyileştirmek için ayırmak istersen o zaman newline kullanılabilir. Burada zaten satırlar bir satır. Bunları newlinelarla ayırmaya gerek yok. Boşuna bu dosyanın satır sayısını arttırıyor bu tür bir kullanım.

Reply:
Done.